index.js

```diff
@@ -7,6 +7,8 @@ const isNil = require('./src/isNil')
 const isPlainObject = require('./src/isPlainObject')
 const pick = require('./src/pick')
 const pickBy = require('./src/pickBy')
+const omit = require('./src/omit')
+const omitBy = require('./src/omitBy')
 const camelize = require('./src/camelize')
 
 module.exports = {
```

Comment:
I know we don't use typescript but having type definitions help on adaption of open source libraries also on devs that use ts, due to that could you update also index.d.ts with new methods please?